src/components/frame/v5/pages/UserAdvancedPage/partials/FeesForm/hooks.tsx

```diff
@@ -1,64 +1,85 @@
-import { bool, object } from 'yup';
+import { bool, object, InferType } from 'yup';
 import { toast } from 'react-toastify';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { UseFormReturn } from 'react-hook-form';
 
 import Toast from '~shared/Extensions/Toast';
 import { canUseMetatransactions } from '~utils/checks';
 import { useUpdateUserProfileMutation } from '~gql';
 import { useAppContext } from '~hooks';
+import { notMaybe } from '~utils/arrays';
+import noop from '~utils/noop';
+import { omit } from '~utils/lodash';
 
 export const useFeesForm = () => {
   const metatransactionsValidationSchema = object({
     metatransactionsEnabled: bool<boolean>(),
   }).defined();
 
+  const [formRef, setFormRef] = useState<UseFormReturn<
```

Comment:
can we take out the typedefs and the validation schema to a separate file or at least outside the component?

Reply:
Moved to `consts`